frontend/src/shared/form/Password.tsx

```diff
@@ -64,9 +64,11 @@ function Password(props: PasswordProps): ReactElement {
 }
 
 type PasswordProps = {
+  /** string placeholder text for input */
```

Comment:
It may be better to rename the variable if it is not clear

Reply:
These are for intellisense to provide documentation, so when items are hovered over it provides additional detail. I am trying to get in the habit of doing this as it can be helpful for prop types. Let me know what you think.

Reply:
I think it's better to have descriptive variable names such that we don't require any additional comments (comments can quickly become outdated and it's an extra thing to maintain)